lib/elixir_segment_api.ex

```diff
@@ -66,4 +66,6 @@ defmodule SegmentAPI do
 
   defp auth_header,
     do: "Basic #{Base.encode64(Application.get_env(:segment_api, :api_key, "") <> ":")}"
+
+  defp json_library, do: Application.get_env(:segment_api, :json_impl, Jason)
```

Comment:
The README specifies the env should be `json_library` not `json_impl` - `json_library` seems to be the conventional env key to use.

Reply:
doh!

Reply:
Hehe